code/modules/reagents/withdrawal/generic_addictions.dm

```diff
@@ -154,3 +154,46 @@
 	REMOVE_TRAIT(affected_human, TRAIT_NIGHT_VISION, type)
 	var/obj/item/organ/eyes/eyes = affected_human.getorgan(/obj/item/organ/eyes)
 	eyes.refresh()
+
+///Makes you a hypochondriac - I'd like to call it hypochondria, but "I could use some hypochondria" doesn't work
+/datum/addiction/medicine
+	name = "medicine"
+	var/datum/hallucination/fake_alert/hallucin
+
+/datum/addiction/medicine/withdrawal_stage_1_process(mob/living/carbon/affected_carbon, delta_time)
+	. = ..()
+	if(prob(10*delta_time))
+		affected_carbon.emote("cough")
+
+/datum/addiction/medicine/withdrawal_enters_stage_1(mob/living/carbon/affected_carbon)
+	. = ..()
+	var/type = pick(list("not_enough_oxy", "too_much_oxy", "temphot", "tempcold"))
```

Comment:
this is immediately going to give this addiction away if you're playing as a race that doesn't normally encounter the too_much_oxy icon (or the not_enough_oxy icon if you're playing a plasmaman).

Reply:
Are there any other races that this won't work well with? I'll write some snowflake code for plasmamen.

Reply:
you could check the gas requirements of the addictee's lungs and whether or not the addictee has the nobreath, coldimmune, and/or heatimmune traits, I suppose

Reply:
Alright, I added checks for those sorts of things!